miasm2/core/objc.py

```diff
@@ -0,0 +1,1721 @@
+"""
+C helper for Miasm:
+* raw C to Miasm expression
+* Miasm expression to raw C
+* Miasm expression to C type
+"""
```

Comment:
There is so many object and type involved that a kind of summary on "who does what" would be a great thing, especially when we will come back on the code later.